configure.ac

```diff
@@ -1,5 +1,5 @@
 AC_PREREQ([2.61])
-AC_INIT([libjwt], [1.12.0], [https://github.com/benmcollins/libjwt/issues])
+AC_INIT([libjwt], [1.12.0a], [https://github.com/benmcollins/libjwt/issues])
```

Comment:
We are using this versioning internally in our company, but I guess this may be not appropriate for this project, so I can change it if suggested.

Reply:
Yes, please remove this from the PR. Thanks.